### PR TITLE
[Update] Japanese translation of newbs.md

### PR DESCRIPTION
**Pull request by mtei**
## Description

Updated the Japanese translation of newbs.md to 0.9.0.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@umi-umi @shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/newbs.md [hunk 1]
```diff
@@ -2,40 +2,47 @@
...
+## このガイドは私のためにあるのでしょうか？
+
+このガイドは、ソースコードを使ってキーボードのファームウェアを構築したいと考えている人に適しています。
+もしあなたがすでにプログラマーであれば、このプロセスはとても身近で簡単に理解できるでしょう。
+もし、プログラミングをすることに抵抗があるのであれば、代わりに[私たちのオンラインGUI](ja/newbs_building_firmware_configurator.md)を見てみてください。
 
 ## 概要
 
-このガイドには7つの主要なセクションがあります。
+このガイドには 4つの主要なセクションがあります。
```
Comment (shelaf) [inline docs/ja/newbs.md:30]:
```suggestion
このガイドには4つの主要なセクションがあります。
```

File: docs/ja/newbs.md [hunk 2]
```diff
@@ -2,40 +2,47 @@
...
 タスクを達成する方法について疑問がある場合は、[案内を求めることができます](ja/getting_started_getting_help.md)。
 
-## 追加のリソース(英語)
+## 追加のリソース
 
-* [Thomas Baart's QMK Basics Blog](https://thomasbaart.nl/category/mechanical-keyboards/firmware/qmk/qmk-basics/) – 新規ユーザ視点から見た QMK ファームウェアの基本的な使用方法をカバーしたユーザ作成のブログ。
+このガイドの他にも、QMK の学習に役立つリソースがいくつかあります。[学習リソース](ja/newbs_learn_more_resources.md)のページにまとめました。
 
-## 追加のリソース(日本語)
+## Open Source
```
Comment (shelaf) [inline docs/ja/newbs.md:46]:
```suggestion
## オープンソース
```

File: docs/ja/newbs.md [hunk 3]
```diff
@@ -2,40 +2,47 @@
...
 もし作成したキーボードがメカニカルキーボードの場合、実行できる可能性が高いです。
-QMK は[多くの趣味のキーボード](http://qmk.fm/keyboards/)をサポートしているため、もし現在のキーボードで QMK を実行できない場合でも、ニーズに合ったキーボードを見つけるのに問題はないはずです。
+QMK は[多くの趣味のキーボード](http://qmk.fm/keyboards/)をサポートしています。
+現在使用しているキーボードが QMK を実行できない場合、QMK を実行できるキーボードの選択肢はたくさんあります。
+
+## このガイドは私のためにあるのでしょうか？
+
+このガイドは、ソースコードを使ってキーボードのファームウェアを構築したいと考えている人に適しています。
+もしあなたがすでにプログラマーであれば、このプロセスはとても身近で簡単に理解できるでしょう。
+もし、プログラミングをすることに抵抗があるのであれば、代わりに[私たちのオンラインGUI](ja/newbs_building_firmware_configurator.md)を見てみてください。
```
Comment (shelaf) [inline docs/ja/newbs.md:26]:
```suggestion
もし、プログラミングの考え方に抵抗があるのであれば、代わりに[私たちのオンラインGUI](ja/newbs_building_firmware_configurator.md)を見てみてください。
```

**Pull request merged**